pkg/resource/web_acl/hooks.go

```diff
@@ -33,3 +42,351 @@ func stringToStatement[T Statement](cfg *string) (*T, error) {
 
 	return &config, nil
 }
+
+// setLoggingConfiguration populates the WebACL's logging configuration from the
```

Comment:
nit: From the what?

Reply:
ooh i just meant, will fix
```suggestion
// setLoggingConfiguration populates the WebACL's logging configuration
```